toolkit/scss/forms/_summary.scss

```diff
@@ -33,71 +33,86 @@
   padding-left: 75%;
   position: relative;
   z-index: 10;
+  font-weight: bold;
```

Comment:
It won't work here because `bold-19` is not a drop-in replacement for `copy-19` (different padding). Have changed it where possible in 0716073.